docs/doxygen/demos/demos_subpages.txt

```diff
@@ -22,9 +22,9 @@ Messages in this demo are sent at QoS 1, which guarantees at least one delivery
 
 /**
 @page mqtt_demo_plaintext Plaintext Demo
-@brief Uses plaintext sockets to establish a TCP connection, and demonstrates QoS 0 implementation.
+@brief Uses a POSIX socket based transport interface implementation to establish a TCP connection, and demonstrates QoS 0 implementation.
```

Comment:
Please mention about use of POSIX sockets based transport interface implementation in the paragraph

```suggestion
@brief Demo of an MQTT application that uses QoS 0 level of communication with broker over a plain TCP connection with server.
```